command/agent/secure_variable_endpoint_test.go

```diff
@@ -268,15 +278,76 @@ func TestHTTP_SecureVariables(t *testing.T) {
 			// Check for the index
 			require.NotZero(t, respW.HeaderMap.Get("X-Nomad-Index"))
 
-			// Check the variable was created
-			out, err := rpcReadSV(s, sv1.Namespace, sv1.Path)
+			{
+				out, err := rpcReadSV(s, sv.Namespace, sv.Path)
+				require.NoError(t, err)
+				require.NotNil(t, out)
+				require.NotEqual(t, svU.SecureVariableMetadata, out.SecureVariableMetadata)
+
+				svU.CreateIndex, svU.ModifyIndex = out.CreateIndex, out.ModifyIndex
+				svU.CreateTime, svU.ModifyTime = out.CreateTime, out.ModifyTime
```

Comment:
Maybe there's something weird going on with the scope here that I'm missing, but what do these two lines get us? (Same for the `update-cas` test below.)

Reply:
They follow-on after a NotEqual check above that ensures that I didn't mutate `svU` in the update, then updating the time and indexes lets me do a simple equality check to make sure the remainder of the metadata is correct.  Probably overkill at the moment, but I'm imagining a time when metadata has more in it than namespace, path.